go-controller/pkg/config/config.go

```diff
@@ -1154,6 +1156,11 @@ var OVNGatewayFlags = []cli.Flag{
 		Usage:       "Disable SNAT for egress traffic with multiple gateways.",
 		Destination: &cliConfig.Gateway.DisableSNATMultipleGWs,
 	},
+	&cli.BoolFlag{
+		Name:        "disable-forwarding-on-interfaces",
+		Usage:       "Enabling forwarding on OVNK controlled interfaces.",
```

Comment:
```suggestion
		Usage:       "Disable forwarding on OVNK controlled interfaces.",
```

Reply:
ah good catch, its because I initially wanted to do en enable-forwarding and then went with disable hehe. will change this in the next iteration.